standalone/index.html

```diff
@@ -0,0 +1,83 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Standalone Sandbox Page!</title>
+	<script type="text/javascript">
+		window.wp = {};
+
+		// Necessary for wp.date
+		window._wpDateSettings = {
```

Comment:
I don't think you need it anymore. We provide defaults in `@wordpres/date`:
https://github.com/WordPress/gutenberg/blob/master/packages/date/src/index.js#L85-L87
where `settings` is locally defined:
https://github.com/WordPress/gutenberg/blob/master/packages/date/src/index.js#L8-L24
and can be overriden using `setSettings` method:
https://github.com/WordPress/gutenberg/blob/master/packages/date/src/index.js#L31